test/integration/ius/inspec/ius_spec.rb

```diff
@@ -2,7 +2,7 @@
   it { should_not be_installed }
 end
 
-describe package('git224') do
+describe package('git236') do
```

Comment:
You need to fix the test below too please.

Reply:
Updated the test for git version on git236